libs/remix-debug/src/source/astWalker.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
```

Comment:
This file was removed to use `remix-astwalker` npm package. Is there any need to bring it back?